src/ooni/utils.cpp

```diff
@@ -0,0 +1,57 @@
+// Part of measurement-kit <https://measurement-kit.github.io/>.
+// Measurement-kit is free software. See AUTHORS and LICENSE for more
+// information on the copying conditions.
+
+#include "src/ooni/utils.hpp"
+
+namespace mk {
+namespace ooni {
+
+ErrorOr<Json> geoip(std::string ip, std::string path_country,
+                    std::string path_asn) {
+    Json json;
+    GeoIP *gi;
+    GeoIPLookup gl;
```

Comment:
Please, add a comment stating that we know that opening the database every time *may* be inefficient and that, in such case, we will consider keeping it open.